File: 백준/Bronze/10798. 세로읽기/세로읽기.py
arr = [[""] * 15 for i in range(5)]

for i in range(5):
    s = input()
    for j in range(len(s)):
        arr[i][j] = s[j]

for j in range(15):
    for i in range(5):
        if arr[i][j] == "":
            continue
        print(arr[i][j],end="")

File: 백준/Bronze/10798. 세로읽기/세로읽기.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {


    var arr = Array(5) {Array(15){""}}

    repeat(5) {
        i ->
        val input = readLine()
        repeat(input.length) {
            j ->
            arr[i][j] = input[j].toString()
        }
    }

    var sb = StringBuilder()

    repeat(15) {
        j ->
        repeat(5) {
            i ->
            sb.append(arr[i][j])
        }
    }
    println(sb.toString())








}